template-tanstack-start-clerk/package.json

```
@@ -9,7 +9,7 @@
     "dev": "concurrently -r npm:dev:web npm:dev:db",
     "dev:web": "vinxi dev",
     "dev:db": "npx convex dev",
-    "build": "vinxi build 2>/dev/null & true 'Ignore vinxi build warnings until tanstack start can run without them (need updated guide)'",
+    "build": "vinxi build 2>/dev/null && true 'Ignore vinxi build warnings until tanstack start can run without them (need updated guide)'",
```

To ignore warnings we need to use `||` here instead of `&&`

```suggestion
    "build": "vinxi build 2>/dev/null || true 'Ignore vinxi build warnings until tanstack start can run without them (need updated guide)'",
```